api/docker-compose.yaml

```diff
@@ -0,0 +1,38 @@
+services:
+  api:
+    profiles: [development]
+    build: .
+    ports:
+      - "5001:5000"
+    volumes:
+      - .:/app
+    env_file:
+      - .env
+    environment:
+      - FLASK_APP=app.py
+      - FLASK_ENV=development
+    command: flask run --host=0.0.0.0 --port 5000
+
+  api-gunicorn:
+    profiles: [production, staging]
+    command:
+      [
+        "gunicorn",
+        "--threads",
+        "4",
+        "--workers",
+        "2",
+        "--bind",
+        "0.0.0.0:5000",
+        "app:app",
+      ]
+    working_dir: /root/app
+    container_name: moped-api-gunicorn
+    build: .
+    restart: unless-stopped
+    ports:
+      - "0.0.0.0:5001:5000"
+    volumes:
+      - .:/root/app
+    env_file:
+      - .env
```

Comment:
do we need an env_template, and I cant remember are the env variables just defined directly in AWS?